frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -526,6 +554,17 @@ class NativeQueryEditor extends Component {
 
 export default _.compose(
   ExplicitSize(),
+  Questions.loadList({
+    // At the moment I'm loading all questions, but ideally I'd only load questions that are tagged in the query, like this:
+    // query: (state, { query }) => {
+    //     const templateTags = query.templateTagsWithoutSnippets();
+    //     const referencedQuestionIds = templateTags.map(tag => tag["card-id"]);
+    //   return {
+    //     ids: referencedQuestionIds,
+    //   };
+    // },
+    loadingAndErrorWrapper: false,
+  }),
```

Comment:
@metabase/frontend-developers How would I implement this? As far as I can see, there's no other example of loading a list of entities by a list of ids. The questions need to be loaded so the completer above can look up their columns.

Reply:
I could create a new endpoint on the backend, e.g. `api/questions/:ids`, and change `Questions.api.list` to switch to use this endpoint if there's a list of ids, similar to how tables allows getting all the tables for a database (below). Is there a better way to fetch multiple entities at once like this?
https://github.com/metabase/metabase/blob/598dc8d8b5ecb0fe6094671648866d976d605ce1/frontend/src/metabase/entities/tables.js#L57-L59